docs/guides/steth-integration-guide.md

```diff
@@ -128,6 +128,14 @@ shares2mint = --------------------------------------------------------------
                 (newTotalPooledEther * 10000) - (feeBasis * _totalRewards)
 ```
 
+### How to get APR?
+
+With withdrawals enabled, the APR calculation method for Lido has changed significantly.
+When Lido V2 protocol finalizes withdrawal requests, the Lido contract excludes funds from TVL and assigns to burn underlying locked requests’ stETH shares in return. In other words, withdrawal finalization decreases both TVL and total shares.
+Old V1 formula isn’t suitable anymore because it catches TVL changes, but skips total shares changes.
```

Comment:
Old V1 approach and formula (https://docs.lido.fi/contracts/lido-oracle#add-calculation-of-staker-rewards-apr) ...